themes/openy_themes/openy_rose/scss/paragraphs/_paragraphs.scss

```diff
@@ -0,0 +1,753 @@
+        font-size: 34px;
+        line-height: 46px;
+        color: $white;
+        @include breakpoint (768px) {
+          font-size: 54px;
+          line-height: 54px;
+        }
+        @include breakpoint (1024px) {
+          font-size: 74px;
+          line-height: 74px;
+        }
+      }
+      &.left {
+        text-align: right;
+        top: 50%;
+        left: 135px;
+        margin-top: -23px;
+        @include breakpoint (768px) {
+          left: 10px;
+          margin-top: -27px;
+        }
+        @include breakpoint (1024px) {
+          margin-top: -37px;
+        }
+        .fa {
+          margin-right: 5px;
+        }
+        span {
+          @include breakpoint($desktop-small) {
+            margin-left: -30px;
+          }
+        }
+      }
+      &.right {
+        text-align: left;
+        top: 50%;
+        right: 135px;
+        margin-top: -23px;
+        @include breakpoint (768px) {
+          right: 10px;
+          margin-top: -27px;
+        }
+        @include breakpoint (1024px) {
+          margin-top: -37px;
+        }
+        .fa {
+          margin-left: 5px;
+        }
+      }
+      span {
+        font-size: 50px;
+        text-shadow: 0 1px 2px rgba(0, 0, 0, 0.6);
+        @include breakpoint(0 $desktop-small) {
+          font-size: 30px;
+        }
+      }
+    }
+    .carousel-inner > .item img,
+    .carousel-inner > .item a img {
+      height: auto;
+      width: 100%;
+    }
+  }
+}
+
+.text-banner {
+  h1 {
+    font-size: 48px;
+    text-transform: uppercase;
+    font-weight: bold;
+    line-height: 1.1em;
+    margin: 20px 0 10px;
+    color: white;
+    @include cachet();
+    @include breakpoint(0 $tablet-start) {
+      font-size: 40px;
+    }
+    @include breakpoint(0 $tablet) {
+      font-size: 20px;
+    }
+  }
+  p {
+    font-size: 20px;
+    margin: 0 0 10px;
+    color: white;
+  }
+}
+
+#banner {
+  position: relative;
+  min-height: 455px;
+  @include breakpoint(0 991px) {
+    height: auto;
+  }
+  &:before {
+    position: absolute;
+    top: 0;
+    left: 50%;
+    z-index: 4;
+    display: block;
+    visibility: visible;
+    width: 66%;
+    height: 100%;
+    background: $black3;
+    content: '';
+    -webkit-transform: skew(-20deg);
+    -moz-transform: skew(-20deg);
+    -ms-transform: skew(-20deg);
+    transform: skew(-20deg);
+    @include breakpoint($desktop) {
+      left: 55%;
+    }
+    @include breakpoint(0 991px) {
+      display: none;
+    }
+  }
+  .image {
+    position: inherit;
+  }
+  .banner-img {
+    position: absolute;
+    top: 0;
+    left: 0;
+    z-index: 0;
+    width: 62%;
+    height: 100%;
+    background-position: center top;
+    background-size: cover;
+    background-repeat: no-repeat;
+  }
+  .banner-img-mobile {
+    @include breakpoint(0 $tablet) {
+      margin-top: 0;
+      margin-bottom: 0;
+      margin-left: -20%;
+      max-width: none;
+      width: 140%;
+    }
+  }
+  .banner-caption {
+    color: white;
```

Comment:
lets not use hardcoded color names instead please use variables

Reply:
Fixed.

Reply:
Wait... this is for Rose?